.github/workflows/ci.yml

```diff
@@ -62,10 +62,21 @@ jobs:
             - name: Run tests
               run: python -m tox -e flake8_7 -- --onlyfuzz --no-cov -n auto
 
+    check_release:
+        runs-on: ubuntu-latest
+        strategy:
+            fail-fast: false
+        steps:
+            - uses: actions/checkout@v4
+            - name: Set up Python 3
+              uses: actions/setup-python@v5
+            - name: Test changelog & version python tests/test_changelog_and_version.py --ensure-tag
+              run: python3 tests/check_changelog_and_version.py
```

Comment:
Name is inconsistent with command now?

Reply:
oops, looks like a bad copy-paste. fixed